mycore-restapi/src/main/java/org/mycore/restapi/v2/access/MCRRestRequiredPermissionDefaultResolver.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of ***  M y C o R e  ***
+ * See http://www.mycore.de/ for details.
+ *
+ * MyCoRe is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MyCoRe is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.mycore.restapi.v2.access;
+
+import jakarta.ws.rs.container.ContainerRequestContext;
+
+public class MCRRestRequiredPermissionDefaultResolver implements MCRRestRequiredPermissionResolver {
+
+    @Override
+    public String resolvePermission(ContainerRequestContext requestContext) {
+        return "";
```

Comment:
should always throw a InternalServerErrorException and the class should therefor be named MCRRestFailingPermissionResolver or similar